Networking/WebSockets/WebSocketInterface.hh

```diff
@@ -83,8 +83,9 @@ namespace litecore { namespace websocket {
         kNetErrHostUnreachable,
         kNetErrAddressNotAvailableAIL,
         kNetErrBrokenPipe,
+        kC4NetErrUnknownInterface,
```

Comment:
Here, by convention, we drop C4 in the name, but double check it gets the number as the C4 one.